Source/SwiftLintFramework/Rules/Style/FileTypesOrderRule.swift

```diff
@@ -125,6 +142,12 @@ public struct FileTypesOrderRule: ConfigurationProviderRule, OptInRule {
         }
     }
 
+    private func libraryContentProviderSubstructures(in file: SwiftLintFile) -> [SourceKittenDictionary] {
+        return file.structureDictionary.substructure.filter { substructure in
+            return substructure.inheritedTypes.contains("LibraryContentProvider")
```

Comment:
None of the the `return`s is required. 😉

Reply:
I kept it because it was in the surroundings, I removed it where it was not necessary